Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN mkdir ${HOME}/.ssh && chmod go-rwx ${HOME}/.ssh \
 ENV PYTHONPATH="${PYTHONPATH}:${HOME}"
 ENV PATH="/home/usr/.local/bin:${PATH}"
 
+FROM base as deps
 # Copy and install Unstructured
 COPY requirements requirements
 
```

Comment:
looks like the `pip install` below this didn't get cached from my end, I always have a step #14 runnning in my docker build log

Reply:
okay its because `--no-cache-dir `